generated/nidaqmx/constants.py

```diff
@@ -962,7 +986,7 @@ class UsageTypeAI(Enum):
     ROSETTE_STRAIN_GAGE = 15980  #: Strain measurement using a rosette strain gage.
     ACCELERATION_ACCELEROMETER_CURRENT_INPUT = 10356  #: Acceleration measurement using an accelerometer.
     ACCELERATION_CHARGE = 16104  #: Acceleration measurement using a charge-based sensor.
-    ACCELERATION_4_WIRE_DC_VOLTAGE = 16106  #: Acceleration measurement using a 4 wire DC voltage based sensor.
+    ACCELERATION_FOUR_WIRE_DC_VOLTAGE = 16106  #: Acceleration measurement using a 4 wire DC voltage based sensor.
```

Comment:
I think I like this?

Reply:
Updated enum values as in expected constants.py